crates/ide/src/references/rename.rs

```diff
@@ -980,7 +1026,7 @@ mod fo$0o;
 "#,
             expect![[r#"
                 RangeInfo {
-                    range: 4..7,
+                    range: 0..0,
                     info: SourceChange {
                         source_file_edits: {
                             FileId(
```

Comment:
For the future, the tests here should use a more visual repr than a Debug. Perhaps printing changed lines or somethning like that? Or specyting the changes in comments, a-la `// ^^^`